packages/wdio-appium-service/package.json

```
@@ -27,6 +27,7 @@
   },
   "dependencies": {
     "@wdio/logger": "6.6.0",
+    "@wdio/config": "^6.6.0",
```

To ensure we reference always the same packages we should pin the versions. This is how Lerna recommends sub package consumption.

```suggestion
    "@wdio/config": "6.6.0",
```